src/modules/minigame/wind_platformer/wind_platformer.tscn

```diff
@@ -64,17 +64,10 @@ initial_rect = Rect2(0, 200, 1920, 1080)
 left_rect = Rect2(-150, 150, -140, 1080)
 right_rect = Rect2(2000, 150, 2010, 1080)
 
-[node name="SharedComponents" parent="." node_paths=PackedStringArray("minigame_node") instance=ExtResource("5_k4m21")]
-minigame_node = NodePath("..")
+[node name="MinigameSharedComponents" parent="." instance=ExtResource("5_k4m21")]
 
-[node name="SharedBaseComponents" parent="SharedComponents" index="1"]
+[node name="SharedBaseComponents" parent="MinigameSharedComponents" index="1"]
```

Comment:
I set the name of the component to be the same as the file everywhere. It's long, but it's the actual name rather than a per-minigame custom name, and less likely to confuse people I think.